### ISSUE TITLE
Build errors in VisualStudio because of missing target framework

### ISSUE THREAD

**Issue by chrischu** [sschmid/Entitas-CSharp#22]
Entitas.Migration project misses target framework which can lead to build errors, since VS can decide which framework it uses and apparently it defaults to 4.0.

**Comment by sschmid** [comment]
The target framework of Entitas.Migration was intentionally set to Mono / .NET 4.0, because when running the command line program, it complained about
```
WARNING: The runtime version supported by this application is unavailable.
Using default runtime: v4.0.30319
```
When I change it to Mono / .NET 4.0, I don't get this warning.
What exactly is VS complaining about? You still should be able to build, or not?

**Comment by chrischu** [comment]
I am not able to build since the Tests project references the Entitas.Migration project and that does not work since the Tests project is using the 3.5 framework.

**Comment by sschmid** [comment]
Ok, that's bad. Merged #23

**Comment by chrischu** [comment]
Thanks :).
